constraints.txt

```diff
@@ -0,0 +1,6 @@
+certifi==2018.4.16
+chardet==3.0.4
+idna==2.6
+nose==1.3.7
```

Comment:
This seems wrong, you might need to re-generate this after adding `nose2`

Reply:
Oops you're right. Edited that commit.